src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -377,12 +375,24 @@ export class StudyViewPageStore {
             };
         }
 
-        _chartMetaSet[UniqueKey.MUTATION_COUNT_CNA_FRACTION] = {
-            uniqueKey: UniqueKey.MUTATION_COUNT_CNA_FRACTION,
-            chartType: ChartType.SCATTER,
-            displayName: 'Mutation count Vs. CNA',
-            description: ''
-        };
+        const scatterRequiredParams = _.reduce(this.clinicalAttributes.result, (acc, next) => {
+            if (MUTATION_COUNT === next.clinicalAttributeId) {
+                acc[MUTATION_COUNT] = true
+            }
+            if (FRACTION_GENOME_ALTERED === next.clinicalAttributeId) {
+                acc[FRACTION_GENOME_ALTERED] = true
+            }
+            return acc;
+        }, {MUTATION_COUNT: false, FRACTION_GENOME_ALTERED: false});
```

Comment:
you need to do `[MUTATION_COUNT]:false, [FRACTION_GENOME_ALTERED]:false` to use the constants

Reply:
Ah good catch.